docs/docs-content/clusters/data-center/vmware/permissions.md

```diff
@@ -7,186 +7,231 @@ sidebar_position: 60
-| **spectro-templates** | Read only. This is the vSphere folder created during the install. For airgap installs, you must manually create this folder.      |
-| **Storage views**     | View                                                                                                                              |
-| **Tasks**             | Create task<br />Update task                                                                                                      |
-| **vApp**              | Import<br />View OVF environment<br />Configure vAPP application<br />Configure vApp instance                                     |
-| **vSphere tagging**   | Assign or Unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag                              |
-
-The following table lists spectro role privileges for VMs by category. All privileges are for the vSphere object,
+| **Category**              | **Privileges**                                                                                                                                                        |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **CNS**                   | Searchable                                                                                                                                                            |
+| **Datastore**             | Allocate space<br />Browse datastore<br />Low level file operations<br />Remove file<br />Update virtual machine files<br />Update virtual machine metadata           |
+| **Folder**                | Create folder<br />Delete folder<br />Move folder<br />Rename folder                                                                                                  |
+| **Host Local Operations** | Reconfigure virtual machine                                                                                                                                           |
+| **Network**               | Assign network                                                                                                                                                        |
+| **Resource**              | Apply recommendation<br />Assign virtual machine to resource pool<br />Migrate powered off virtual machine<br />Migrate powered on virtual machine<br />Query vMotion |
+| **Sessions**              | Validate session                                                                                                                                                      |
+| **spectro-templates**     | Read only. This is the vSphere folder created during the install. For airgap installs, you must manually create this folder.                                          |
```

Comment:
I think this is a typo as this isn't a permission but an entity on which to apply permission

Reply:
Resolved in eff3548d1103ad25895211ffc3fc1076aabc3de1